index.rst

```diff
@@ -222,25 +239,37 @@ If Science Platform administrators need to test pod creation or see the event st
     To monitor the status of the pod creation, use ``GET /nublado/spawner/v1/labs/<username>/events``.
 
     The body of the ``POST`` request is a specification for the lab.
+
```

Comment:
This is an example of the body, so that was supposed to be all one paragraph and I think it still reads a bit better that way.